seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigBuilder.java

```diff
@@ -94,9 +99,57 @@ public static Config of(@NonNull Map<String, Object> objectMap, boolean isEncryp
         if (!isEncrypt) {
             config = ConfigShadeUtils.decryptConfig(config);
         }
+        log.info("Parsed config file: \n{}", mapToJson(privacyHandler(config.root().unwrapped())));
         return config;
     }
 
+    public static Map<String, Object> privacyHandler(Map<String, Object> configMap) {
+        List<String> logPrivacyKeyword = seaTunnelConfig.getEngineConfig().getLogPrivacyKeyword();
```

Comment:
Please just use https://github.com/apache/seatunnel/blob/16950a67cd5990ed3510fd91564390e2ab2cd44c/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/ConfigShadeUtils.java#L50 as value for now.

Reply:
@hawk9821